tests/test_user.py

```diff
@@ -51,3 +51,12 @@ def test_signed_in_user_can_open_stations_page(authenticated_browser):
 
     stations_page_title = stations_page.get_page_title()
     assert stations_page_title == 'Stations'
+
+    welcome_user_message = stations_page.get_welcome_user_message()
+    assert 'welcome, ' in welcome_user_message
+
+    # TODO - replace hardcoded names below with actual data from the db
```

Comment:
It appears that to test Django models I will need:
* [pytest-django](https://pytest-django.readthedocs.io/en/latest/) module, or
* the following 'work-around' (found on [SO](https://stackoverflow.com/a/27941367)):

```
import django
import os
os.environ['DJANGO_SETTINGS_MODULE'] = 'jcc.settings.development'
django.setup()
```